business_objects/project.py

```diff
@@ -42,6 +45,31 @@ def get_all(organization_id: str) -> List[Project]:
     )
 
 
+def get_all_by_user(user_id: str) -> List[Project]:
+    organization_id = (
+        session.query(User).filter(User.id == user_id).first().organization_id
+    )
+
+    projects = (
+        session.query(Project).filter(Project.organization_id == organization_id).all()
+    )
+
+    project_dicts = sql_alchemy_to_dict(projects)
+
+    for project in project_dicts:
+        user_id = project["created_by"]
+        mail = kratos.resolve_user_mail_by_id(user_id)
+        names = kratos.resolve_user_name_by_id(user_id)
```

Comment:
kratos import can't be used in the submodule as not all repos using the submodule can access it.
Usually we collect db data in the submodule and if necessary extend it in the manager files